sbt_common/display/src/main/scala/uk/ac/wellcome/display/models/DisplayWork.scala

```diff
@@ -24,6 +24,10 @@ case class DisplayWork(
     dataType = "String",
     value = "A description of specific physical characteristics of the work.") physicalDescription: Option[
     String] = None,
+  @ApiModelProperty(
+    dataType = "List[uk.ac.wellcome.display.models.WorkType]",
+    value = "The type of work.") workType: Option[
```

Comment:
@jtweed Can you advise on that this should _really_ be?

Reply:
FWIW, I usually use whatever’s in the Work ontology.

Reply:
What @alexwlchan said.